### PR TITLE
1. added `version` command for client memcached. why not)

### PR DESCRIPTION
**Pull request by ArtemIsmagilov**
1. added version command(https://github.com/memcached/memcached/blob/9dc8b148345b27dd86580641d17f4268885b5894/doc/protocol.txt#L1823)
2. added tests
3. -U some actions and requirements

### CODE REVIEW

File: emcache/client.py [hunk 1]
```diff
@@ -623,6 +623,30 @@ async def flush_all(
 
         return
 
+    async def version(self, memcached_host_address: MemcachedHostAddress) -> Optional[Item]:
```
Comment (pfreixes) [inline emcache/client.py:626]: I would go for returning just an `Optional[str]`, `Item` data type is reserved for key values.

File: tests/acceptance/test_other_commands.py [hunk 2]
```diff
@@ -162,3 +162,9 @@ async def test_flush_all_with_delay(self, client, key_generation, memcached_addr
         # item should not be found.
         item = await client.get(key_and_value)
         assert item is None
+
+
+class TestVersion:
+    async def test_version(self, client, memcached_address_1, memcached_address_2):
+        assert await client.version(memcached_address_1)
```
Comment (pfreixes) [inline tests/acceptance/test_other_commands.py:169]: Can we assert the version value too?

File: tests/unit/test_protocol.py [hunk 3]
```diff
@@ -240,6 +240,21 @@ async def coro():
...
+            return await protocol.version_command()
+
+        task = event_loop.create_task(coro())
+        await asyncio.sleep(0)
+
+        protocol.data_received(b"VERSION 1.6.26\r\n")
+
+        result = await task
+
+        assert result.startswith(b"VERSION")
```
Comment (pfreixes) [inline tests/unit/test_protocol.py:254]: I believe that we can assert the full text here. isn't it?

**Review by pfreixes — CHANGES_REQUESTED** [review]
Thanks for the contribution, just three minor comments!

File: tests/unit/test_protocol.py (hunk 3 above)
Comment (ArtemIsmagilov) [reply tests/unit/test_protocol.py:254]: I think that the client may have different versions of memcached - https://github.com/memcached/memcached/blob/9dc8b148345b27dd86580641d17f4268885b5894/doc/protocol.txt#L1829

File: tests/acceptance/test_other_commands.py (hunk 2 above)
Comment (ArtemIsmagilov) [reply tests/acceptance/test_other_commands.py:169]: you are right, we need to check that the expected response is returned

**Comment by ArtemIsmagilov** [comment]
I'm preparing the next commit with fixes)

File: emcache/client.py (hunk 1 above)
Comment (ArtemIsmagilov) [reply emcache/client.py:626]: Of course, thanks, we'll redo it too)

File: tests/unit/test_protocol.py (hunk 3 above)
Comment (ArtemIsmagilov) [reply tests/unit/test_protocol.py:254]: I think there may be different versions of memcached. Correct me if I don't see something obvious

File: emcache/client.py (hunk 1 above)
Comment (ArtemIsmagilov) [reply emcache/client.py]: +

**Comment by ArtemIsmagilov** [comment]
need squash commit for PR? Maybe it will be more convenient

**Comment by pfreixes** [comment]
> need squash commit for PR? Maybe it will be more convenient
I can squash it during the merge, don't worry

File: tests/unit/test_protocol.py (hunk 3 above)
Comment (pfreixes) [reply tests/unit/test_protocol.py:254]: Oks let me know, once this is changed the PR will be ready to be merged.

Comment (ArtemIsmagilov) [reply tests/unit/test_protocol.py]: Well, changes is completed. You can check)

**Review by pfreixes — APPROVED** [review]

**Comment by ArtemIsmagilov** [comment]
Сool, thank you for providing the opportunity to contribute to open source, quick answers and detailed comments) I’m glad that I can be useful)

**Pull request merged**